xep-0403.xml

```diff
@@ -207,64 +254,143 @@ A user MAY share presence information with the channel, for one or more online c
 
       <example caption="Channel Distributes Notification of Client going Offline">
-        <![CDATA[<presence from='12345#coven@mix.shakespeare.example/UUID-a1j/7533'
+<![CDATA[<presence from='12345#coven@mix.shakespeare.example/UUID-a1j/7533'
           to='hecate@shakespeare.example'
           id='77E07BB0-55CF-4BD4-890E-3F7C0E686BBD'
-          type='unavailable'/>]]></example>
+          type='unavailable'>
+      <mix xmlns='urn:xmpp:presence:0'>
+        <jid>hecate@shakespeare.example/UUID-x4r/2491</jid>
+        <nick>thirdwitch</jid>
+      </mix>
+</presence>]]></example>
 
       <p>
         There is the possibility that the message associated with the user going offline will be lost.   If this happens, "ghost" entries will appear in the presence node.   A MIX service MAY take steps to address this, for example by probing client with a disco for presence items that remain unchanged for a long period.
       </p>
 
 
 
-    </section3>
+    </section2>
 
-    <section3 topic="User Leaving a Channel" anchor="usecase-presence-leave">
+    <section2 topic="User Leaving a Channel" anchor="usecase-presence-leave">
       <p>
         The primary actions for a user leaving a channel are specified in &xep0369;.    This section sets out additional actions for handling presence.    When a user leaves the channel, all entries for the user's clients MUST be removed from the participants node.   The MIX channel MUST distribute unavailable presence notifications for each client removed to all subscribers of the participants node.
       </p>
 
       <example caption="Channel Distributes Notification when User Leaves Channel">
-        <![CDATA[<presence from='12345#coven@mix.shakespeare.example/UUID-a1j/7533'
+<![CDATA[<presence from='12345#coven@mix.shakespeare.example/UUID-a1j/7533'
           to='hecate@shakespeare.example'
           id='77E07BB0-55CF-4BD4-890E-3F7C0E686BBD'
-          type='unavailable'/>]]></example>
+          type='unavailable'>
+      <mix xmlns='urn:xmpp:presence:0'>
+        <jid>hecate@shakespeare.example/UUID-x4r/2491</jid>
+        <nick>thirdwitch</jid>
+      </mix>
+</presence>]]></example>
 
+    </section2>
 
-    </section3>
 
+<section2 topic="Relaying IQ Stanzas" anchor="usecase-iq-relay">
+  <p>
+    MIX channels MAY relay IQ stanzas between participants.   This is often useful to obtain client information where a direct request to the client would be blocked.   When a client sends an IQ stanza through a MIX channel, it will set the 'from' to its own JID and set the 'to' to the encoded JID of the recipient.    The MIX channel will modify the JIDs in the outgoing message, so that the 'to' is the full JID of the recipient and the 'from' is the encoded JID of the sender.   This is illustrated in the vCard section below.
```

Comment:
I think this elides that there is both bare-JID (which should probably be in 369) and full-JID routing.